desktop/apps/consign/routes.js

```diff
@@ -39,7 +46,34 @@ export const landingPage = async (req, res, next) => {
 }
 
 export const submissionFlow = async (req, res, next) => {
-  res.render('submission_flow', { user: req.user })
+  const store = configureStore(reducers, {
+    submissionFlow: appInitialState
+  })
+  const context = {}
+
+  const layout = renderReactLayout({
+    basePath: res.app.get('views'),
+    mainLayout: 'desktop/apps/consign/templates/layout.jade',
+    blocks: {
+      head: 'meta.jade',
+      body: props => (
+        <StaticRouter
+          location={req.url}
+          context={context}>
+          <App store={store} {...props} />
+        </StaticRouter>
+      )
+    },
+    locals: {
+      ...res.locals,
+      assetPackage: 'misc',
+      bodyClass: 'consignments-submission-body minimal-header body-header-fixed'
+    },
+    data: {
+    }
+  })
+
```

Comment:
Yup! server routing works great.
I still need to fix routing when using `<BrowserRouter />` though. It works fine when we use `<Link ... />` but not when routing via redux actions. I'll update the PR with that fix as soon as I get to work though